attestation/commandrun/tracing_linux.go

```diff
@@ -142,84 +191,279 @@ func (p *ptraceContext) nextSyscall(pid int) error {
 	return nil
 }
 
-func (p *ptraceContext) handleSyscall(pid int, regs unix.PtraceRegs) error {
-	argArray := getSyscallArgs(regs)
-	syscallId := getSyscallId(regs)
+func (p *ptraceContext) handleExeCve(pid int, argArray []uintptr) error {
```

Comment:
breaking these out into their own functions is very welcome change. I don't think the `C` in `ExeCve` should be capitalized, though